api/config.py

```diff
@@ -209,6 +209,7 @@ def __init__(self):
         self.CELERY_RESULT_BACKEND = 'db+{}'.format(self.SQLALCHEMY_DATABASE_URI) \
             if self.CELERY_BACKEND == 'database' else self.CELERY_BROKER_URL
         self.BROKER_USE_SSL = self.CELERY_BROKER_URL.startswith('rediss://')
+        self.CELERY_GLOBAL_PREFIX = get_env("CELERY_GLOBAL_PREFIX")
```

Comment:
The default configuration of `global_keyprefix` is an empty string '', please set the default value of CELERY_GLOBAL_PREFIX to an empty string in DEFAULTS of config.py.